src/components/ResponsiveList/ResponsiveList.jsx

```diff
@@ -74,12 +80,21 @@ const ResponsiveList = forwardRef(
                 ariaLabel={menuButtonAriaLabel}
                 {...menuButtonProps}
               >
-                <div className="responsive-list-menu-wrapper-flex">{menuChildren}</div>
+                <div className={cx(styles.menuWrapperFlex, "responsive-list-menu-wrapper-flex")}>{menuChildren}</div>
               </MenuButton>
             )}
           </div>
         )}
-        <div ref={mergedRef} className={cx("responsive-list--wrapper responsive-list--dummy", className)}>
+        <div
+          ref={mergedRef}
+          className={cx(
+            styles.responsiveList,
+            "responsive-list--wrapper",
+            styles.dummy,
```

Comment:
don't see `dummy` style, I see only `responsiveListDummy` - should be renamed in one of the places
TS is not giving an error for it?